### PR TITLE
bluetooth: fast_pair: Describe discoverable advertising limitation

### PR DESCRIPTION
**Pull request by MarekPieta**
User needs to ensure that RPA timeout would not happen while discoverable advertising is active.

Jira: NCSDK-14830

### CODE REVIEW

**Review by kapi-no — APPROVED** [review]

File: doc/nrf/libraries/bluetooth_services/services/fast_pair.rst [hunk 1]
```diff
@@ -47,9 +47,13 @@ Bluetooth privacy
 -----------------
 
 The service selects Bluetooth privacy (:kconfig:option:`CONFIG_BT_PRIVACY`) and sets the default value of :kconfig:option:`CONFIG_BT_RPA_TIMEOUT` to 3600 seconds.
+
 During not discoverable advertising, the RPA address rotation must be done together with the Fast Pair payload update.
 Because of this, the RPA address cannot be rotated by Zephyr in the background.
 
+During discoverable advertising session, the RPA address rotation must not happen.
+Make sure that your advertising session is shorter than :kconfig:option:`CONFIG_BT_RPA_TIMEOUT` and call :c:func:`bt_le_oob_get_local` to trigger RPA rotation and reset the RPA timeout right before discoverable advertising start.
```
[the remark this answers is not in the document]
Comment (MarekPieta) [reply doc/nrf/libraries/bluetooth_services/services/fast_pair.rst:55]: I think we are mixing things a bit with new wording.
User actually needs to `Make sure that your advertising session is shorter than :kconfig:option:`CONFIG_BT_RPA_TIMEOUT` and call :c:func:`bt_le_oob_get_local` to trigger RPA rotation and reset the RPA timeout right before discoverable advertising start.` in order to `prevent RPA rotation from happening`

[the remark this answers is not in the document]
Comment (MarekPieta) [reply doc/nrf/libraries/bluetooth_services/services/fast_pair.rst:55]: I would not make this as separate points because of that.

Comment (divipillai) [inline doc/nrf/libraries/bluetooth_services/services/fast_pair.rst:55]: My bad, I misunderstood it. I have re-phrased the sentences.
```suggestion
During discoverable advertising session, the RPA address rotation must not happen.
Therefore, consider the following points:

* Make sure that your advertising session is shorter than the value in the :kconfig:option:`CONFIG_BT_RPA_TIMEOUT` option.
* Call :c:func:`bt_le_oob_get_local` to trigger RPA rotation.
* Reset the RPA timeout right before the start of advertising.
```

**Comment by MarekPieta** [comment]
@divipillai I rephrased text a bit (:c:func:`bt_le_oob_get_local` does both mentioned tasks) and added explanation what RPA is. Please rereview

**Review by divipillai — APPROVED** [review]

File: doc/nrf/ug_bt_fast_pair.rst [hunk 2]
```diff
@@ -85,7 +85,10 @@ The Fast Pair service implementation provides API to generate the advertising da
   This function is to be used to set pairing mode before the advertising is started.
 
 Since you control the advertising, make sure to use advertising parameters consistent with the specification.
-The Bluetooth privacy is selected by the Fast Pair service, but you must make sure that the private address rotation is synchronized with the advertising payload update during the not discoverable advertising.
+The Bluetooth privacy is selected by the Fast Pair service, but you must make sure that:
```
Comment (greg-fer) [inline doc/nrf/ug_bt_fast_pair.rst:88]:
```suggestion
The Bluetooth privacy is selected by the Fast Pair service, but you must make sure that the following points are addressed:
```

**Review by greg-fer — APPROVED** [review]
Approving, but please take care of my single comment.

File: doc/nrf/ug_bt_fast_pair.rst (hunk 2 above)
Comment (MarekPieta) [reply doc/nrf/ug_bt_fast_pair.rst:88]: Added: `... the following requirements are met` (I think these are not changes). Please rereview

**Review by greg-fer — APPROVED** [review]

**Pull request merged**